src/components/Inputs/Input.types.ts

```diff
@@ -212,6 +211,17 @@ export interface InputProps<T>
         OcBaseProps<T>,
         'onChange' | 'onFocus' | 'onBlur' | 'onKeyDown'
     > {
+    /**
+     * The input icon alignment.
+     * @default TextInputIconAlign.Left
+     */
+    alignIcon?: TextInputIconAlign;
+    /**
+     * option to show the clear input button.
+     * default is true for backward compatibility
```

Comment:
how many implementations need this? feel like we are early enough in development that we just break it backwards compatibility in this case unless this is desirable in more cases than not.

Reply:
Moving clearable to the root interface should be safe.